content/docs/user-guide/networking/settings.md

```diff
@@ -111,12 +111,13 @@ These settings control how a server will be launched during the "play-in-editor"
 | editorsv_process                   | The file path to your project’s ServerLauncher. If `editorsv_enabled` and `editorsv_launch` is true, the Editor will attempt to launch its own server when entering game mode. By default it looks for the ServerLauncher executable inside the same folder where Editor lives; `editorsv_process` overrides that path. | ""        |                                                                                                                                                  |
 | editorsv_rhi_override              | If `editorsv_launch` is true, the server will use the same render hardware interface (rhi) that the editor is using. For example, if the editor is using DX12, then the new server will be launched using DX12. `editorsv_rhi_override` can be used to override the rhi.                                                 | ""        | If you don't need to see the launched server's graphics then set `editorsv_rhi_override=null`, the null renderer.                                |
 | editorsv_enabled                   | If true the Editor will attempt to connect to a Multiplayer server upon entering game mode.                                                                                                                                                                                                                              | False     ||
-| editorsv_hidden                    | If true the Editor will automatically launch a server upon entering game mode. Set `editorsv_hidden` to true if you want the launched server to be started as a background process without any visible window.                                                                                                           | False     |                                                                                                                                                  |  
+| editor_clientserver                | Whether the Editor should act as a server and a client at the same time, without launching a dedicated server process.  | False | Only applies if `editorsv_enabled` is also true. |
+| editorsv_hidden                    | If true the Editor will automatically launch a server upon entering game mode. Set `editorsv_hidden` to true if you want the launched server to be started as a background process without any visible window.                                                                                                           | False     |                                                                                                                                                  |
```

Comment:
```suggestion
| editorsv_hidden                    | If the Editor is configured to automatically launch a server upon entering game mode, set `editorsv_hidden` to true if you want the launched server to be started as a background process without any visible window.                                                                                                           | False     |                                                                                                                                                  |
```

This description doesn't make sense to me. You may want to include detail on how editorsv_enabled and edtiro_clientserver matter here. A docs maintainer's opinion here would be great.

Reply:
I reverted my whitespace changes. I didn't change any of this in this PR.